File: PCD_CF849_Div4/A_CodeforcesChecking.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int t;
    cin>>t;
    while(t--){
        char c;
        cin>>c;
        string s="codeforces";
        bool check=false;
        for(int i=0; i<10; i++){
            if(s[i]==c){
                check=true;
                break;
            }
        }
        if(check){
            cout<<"YES"<<endl;
        }
        else{
            cout<<"NO"<<endl;
        }
    }
    return 0;
}

File: PCD_CF849_Div4/A_CodeforcesChecking.py
t=int(input())
while t>0:
    s="codeforces"
    c=input()
    check=False
    for i in range(len(s)):
        if s[i]==c:
            check=True
            print("YES")
            break
    if check==False:
        print("NO")
    t-=1